docs/_user_guide/best_practices_jupyter_does_not_work.md

```diff
@@ -1,35 +1,55 @@
 There are several common reasons why JupyterHub might not be working for you.
 
 ## Are you using Safari?
```

Comment:
When you say every word in every title do you mean
Do You Have Access to the Yen Servers? 
or
Do you Have Access To The Yen Servers?

Reply:
the first - "Do You Have Access to the Yen Servers?"

Reply:
I think you can pick a style and use it consistently. ChatGPT is good at this. You can say "APA style guide for title "Do you have access to the yen servers" and it will capitalize for you if you are not sure

Reply:
Thank you